de.sebastianruziczka.looksyk.yml

```diff
@@ -0,0 +1,56 @@
+app-id: de.sebastianruziczka.looksyk
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+# node24 not yet available for 25.08
+base-version: '25.08'
+command: looksyk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node24
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/node24/bin:/usr/lib/sdk/rust-stable/bin
+  env:
+    CARGO_HOME: cargo-cache
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=home
```

Comment:
why does it need all of home now?

Reply:
Electron does not support the file-portal:

https://github.com/electron/electron/issues/30650

and electrons internal file api wants to load it "native"